### PR TITLE
Feat: BNGP-1658 upload-agent

### PR DESCRIPTION
**Pull request by KiranVarma02**

### CODE REVIEW

File: packages/webapp/src/routes/land/upload-consent.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      return h.view(constants.views.LANDOWNER_PERMISSION_UPLOAD, {
+        err: [{
+          text: 'The selected file must be a DOC, DOCX or PDF',
+          href: landownerPermissionUploadId
+        }]
+      })
+    case constants.uploadErrors.maximumFileSizeExceeded:
+      return maximumFileSizeExceeded(h, landownerPermissionUploadId, process.env.MAX_GEOSPATIAL_LAND_BOUNDARY_UPLOAD_MB, constants.views.LANDOWNER_PERMISSION_UPLOAD)
+    default:
+      if (err.message.indexOf('timed out') > 0) {
```
Comment (teddmason) [inline packages/webapp/src/routes/land/upload-consent.js:49]: This is a bug i sorted in the defender ticket, but this should be > -1 not 0.  I fixed the other upload routes in my ticket

**Review by teddmason — DISMISSED** [review]

File: packages/webapp/src/routes/land/upload-consent.js (hunk 1 above)
Comment (KiranVarma02) [reply packages/webapp/src/routes/land/upload-consent.js:49]: done

**Comment by shaundefra** [comment]
Do NOT MERGE THIS BRANCH AS IS redundant https://eaflood.atlassian.net/browse/BNGP-1658?focusedCommentId=540603

**Comment by shaundefra** [comment]
Do NOT MERGE THIS BRANCH AS IS redundant https://eaflood.atlassian.net/browse/BNGP-1658?focusedCommentId=540603

**Pull request closed without merging**